Gemfile

```diff
@@ -102,4 +102,5 @@ gem 'builder'
 
 # run this command to work from a local copy of the gem's repo
 # bundle config local.cmr_metadata_preview /path/to/local/git/repository
-gem 'cmr_metadata_preview', git: 'https://git.earthdata.nasa.gov/scm/cmr/cmr_metadata_preview.git', branch: 'master'
+# gem 'cmr_metadata_preview', git: 'https://git.earthdata.nasa.gov/scm/cmr/cmr_metadata_preview.git', branch: 'master'
+gem 'cmr_metadata_preview', git: 'https://git.earthdata.nasa.gov/scm/cmr/cmr_metadata_preview.git', branch: 'MMT-458'
```

Comment:
Yeah, that's why I thought it was ok to leave the commented line in there as I would change it back before merging.
It's a funny process, but essentially there's no way to prevent running the tests after the preview gem PR gets merged.